src/H5Faccum.c

```diff
@@ -47,6 +47,7 @@
 #define H5F_ACCUM_THROTTLE  8
 #define H5F_ACCUM_THRESHOLD 2048
 #define H5F_ACCUM_MAX_SIZE  (1024 * 1024) /* Max. accum. buf size (max. I/Os will be 1/2 this size) */
+#define H5F_LOC_VALID(x)    (x != HADDR_UNDEF)
```

Comment:
Is there a reason H5F_addr_defined(X) couldn't be used?

Reply:
No at all - except that I wasn't aware of this macro.  I'll change it.

Reply:
Thanks!